docs/ops/generation/Multinomial_13.md

```diff
@@ -0,0 +1,181 @@
+  * For each probability x, replace it with a value e^x
+* Create a cumulative sum of those probabilities, ie. create an array of values where the ith value is the sum of the probabilities x1, ..., xi
+* Divide the created array by its last (maximum) value to normalize the cumulative probabilities between [0, 1]
+* Randomly generate a sequence of floats in the range range [0,1]
+* For each generated number, assign the class with the lowest index for which the cumulative probability is less or equal the generated value.
+* If *with_replacement* is False (sampling without replacement)
+  * Set the class probability to 0 by adjusting the cumulative sum
+  * Assume class with index i has been selected - then every cdf value starting at i-th index should be lowered by the probability of the selected class
+  * Afterwards, divide the cdf by its last (maximum) value to normalize the cumulative probabilities between [0, 1]
+* Convert the output indices to *output_type*
+* Return output indices
+
+**Example computations**:
+Example 1 - 1D tensor
+* Let *input* = ``[0.1, 0.5, 0.4]``, *num_samples* = 5, *log_probs* = false, *replacement* = true
+* CDF of *input* = ``[0.1, 0.1 + 0.5, 0.1 + 0.5 + 0.4]`` = ``[0.1, 0.6, 1]``
+* Randomly generated floats = ``[0.2, 0.4, 0.6, 0.8, 1]``
+* Assigned classes = ``[1, 1, 1, 2, 2]``
+
+Example 2 - 2D tensor, log probabilities
+* Let *input* = ``[[-1, 1, 2], [50, 1, 21]]``, *num_samples* = 10, *log_probs* = true, *replacement* = true
+* Exponentiated *input* = ``[[0.36, 2.71, 7.38], [5184705528587072464087.45, 2.71, 1318815734.48]] 
+* CDF of *input*, per batch = ``[[0.36, 3.07, 10.45], [5184705528587072464087.45, 5184705528587072464090.16, 5184705528588391279824.64]]``
+* Normalized CDF = ``[[0.03, 0.29, 1], [1, 0, 0]]``
+* Randomly generated floats = ``[[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1], [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1]]``
+* Assigned classes = ``[[1, 1, 2, 2, 2, 2, 2, 2, 2, 2], [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]]``
+
+Example 3 - 1D tensor, without replacement
+* Let *input* = ``[0.1, 0.5, 0.4]``, *num_samples* = 2, *log_probs* = false, *replacement* = false
+* CDF of *input* = ``[0.1, 0.6, 1]``
+* Randomly generated floats = ``[0.3, 0.6]``
+* In a loop:
+* For a value of 0.3, class with idx *1* is selected 
+* Therefore, in CDF, for every class starting with idx *1* subtract the probability of class *1* = 0.5
+* CDF = ``[0.1, 0.6 - 0.5, 1.0 - 0.5]`` = ``[0.1, 0.1, 0.5]``
+* Normalize CDF by dividing by last value: CDF = ``[0.2, 0.2, 1.0]``
+* Take the next randomly generated float, here 0.6, and repeat until all random samples have assigned classes
+* Assigned classes = ``[1, 2]``
+
+
+**Attributes**:
+
+* ``output_type``
+
+  * **Description**: the type of the output. Determines generation algorithm and affects resulting values. Output numbers generated for different values of *output_type* may not be equal.
+  * **Range of values**: "i32", "i64", "f16", "bf16", "f32", "f64".
```

Comment:
Please check this PR #19830  regarding removal of f16, bf16, f64 from evaluates.
Maybe it will impact on this value type list.

Reply:
Removed the aforementioned values, updated the spec & ref